ophyd/tests/test_signal.py

```diff
@@ -561,19 +561,34 @@ def test_epicssignal_pv_reuse(cleanup, pvname, count):
         assert len(set(id(sig._read_pv) for sig in signals)) == 1
 
 
-@pytest.mark.parametrize('path',
+@pytest.fixture(scope='function')
+def path_signal(cleanup, signal_test_ioc):
+    sig = EpicsPathSignal(signal_test_ioc.pvs['path'], name='path',
+                          path_semantics='posix')
+    cleanup.add(sig)
+    sig.wait_for_connection()
+    return sig
+
+
+@pytest.mark.parametrize('paths',
                          [('C:\\some\\path\\here'),
-                          ('D:\\here\\is\\another\\')
+                          ('D:\\here\\is\\another\\'),
+                          ('C:/yet/another/path'),
+                          ('D:/more/paths/here/')
                           ])
-def test_windows_paths(path):
-    signal_nt = EpicsPathSignal('TEST', path_semantics='nt')
-    signal_nt.set(path).wait(3)
+def test_windows_paths(paths, cleanup, signal_test_ioc):
```

Comment:
This can use the `path_signal` fixture. It doesn’t need to access `signal_test_ioc` directly or do its cleanup.